templates/pages/studies/show.html

```diff
@@ -134,6 +134,95 @@ <h3>
           </div>
         {% endfor %}
       </div>
+
+      <h2>Calculations</h2>
```

Comment:
The new "Calculations" section mirrors the "Measurements" one, though the "Compare" buttons don't work correctly yet.